### DISCUSSION TITLE
WiRL con Https

### DISCUSSION THREAD

**Discussion by clauxs88-dev** [delphi-blocks/WiRL#52] [General]
Consulta; como trabajo con el componente de modo a que pueda exponer el https en vez del http? estaba probando con la demo DemoOpenAPI.dproj

**Comment by lminuti** [comment]
You can find an instance of `TIdHTTPServer` in `TWiRLServer.ServerImplementation`. But I recommend you to use a web server (apache, ngnix, ..) and set up a reverse proxy to reach the WiRL server

**Reply by clauxs88-dev** [reply]
I was just trying to configure the `TWiRLServer.ServerImplementation` property but I couldn't. Could you give me an example of how to implement the `TWiRLServer.ServerImplementation.IOHandler` for example?

**Reply by lminuti** [reply]
```
  RESTServer := TWiRLServer.Create(Self);

  // WiRL server setup
  // ...
  // RESTServer.AddEngine<TWiRLEngine>('/rest')
  // ...

  LSSLHandler := TIdServerIOHandlerSSLOpenSSL.Create(Self);
  // SSL setup
  // ....
  (RESTServer.ServerImplementation as TIdHTTPServer).IOHandler := LSSLHandler;

```

**Reply by clauxs88-dev** [reply]
Thanks teacher, I'm going to try it!
